acrobat/scripts/dcLana.js

```diff
@@ -1,4 +1,5 @@
 export default function lanaLogging() {
+  const fricPage = window.location.pathname.indexOf('online/') > 0;
```

Comment:
indexOf is kinda old function.
I'd use includes() instead.
```suggestion
  if (window.location.pathname.includes('online/')) {
```
But really, I don't think we want this since it will be `true` for all any other URLs potentially includes `online`. This may result in unwanted result.

Can we actually move this into `frictionless.js` so it gets run only for frictionless?
Like move 
```
const { default: lanaLogging } = await import('./dcLana.js');
lanaLogging();
```
from scripts.js to frictionless.js?

Reply:
it's `online/` not `online`. Everything in the directory has the widget.
I want all the LANA code together